pkgs/servers/home-assistant/custom-components/meshcore_ha/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildHomeAssistantComponent,
+  ruff,
```

Comment:
Why does this need `ruff`? I separately this at https://github.com/NixOS/nixpkgs/compare/master...jpds:ha-meshcore-2.1.5 before knowing this was here (if you want to copy what I have there).